e2e/fixtures/events/claim/defaultJudgmentSpec.js

```diff
@@ -18,26 +17,32 @@ module.exports = {
       currentDefendant: 'Has Sir John Doe paid some of the amount owed?',
       currentDefendantName: 'Sir John Doe',
       businessProcess: {
-        camundaEvent: 'CREATE_CLAIM_SPEC',
+        camundaEvent: 'CREATE_CLAIM_SPEC_AFTER_PAYMENT',
         status: 'FINISHED'
       }
     },
+    showCertifyStatementSpec: {
+      CPRAcceptance: {
+        acceptance: ['CERTIFIED']
+      }
+    },
+    claimPartialPayment : {
+      partialPayment: 'No',
+      showOldDJFixedCostsScreen: 'Yes',
+      currentDefendantName: 'Sir John Doe'
+    },
     paymentConfirmationSpec: {
-      repaymentSummaryObject: 'The judgment will order Sir John Doe to pay £1580.00, including the claim fee and interest, if applicable, as shown:\n### Claim amount \n £1500.00\n### Claim fee amount \n £80.00\n ## Subtotal \n £1580.00\n\n ## Total still owed \n £1580.00'
```

Comment:
Why is this removed?

Reply:
also the config we have in civil service contains data for many different journeys of default judgement, since we only use dj for only our cosc(in this repo), we don't need that many journeys